x-pack/plugins/cloud_security_posture/server/lib/telemetry/collectors/installation_stats_collector.ts

```diff
@@ -8,15 +8,60 @@ import type { CoreStart, Logger, SavedObjectsClientContract } from '@kbn/core/se
 import type { ElasticsearchClient } from '@kbn/core-elasticsearch-server';
 import {
   AgentPolicy,
-  PackagePolicy,
   PACKAGE_POLICY_SAVED_OBJECT_TYPE,
+  PackagePolicy,
   SO_SEARCH_LIMIT,
 } from '@kbn/fleet-plugin/common';
 import { agentPolicyService } from '@kbn/fleet-plugin/server/services';
 import type { CloudSecurityInstallationStats } from './types';
 import type { CspServerPluginStart, CspServerPluginStartDeps } from '../../../types';
 import { CLOUD_SECURITY_POSTURE_PACKAGE_NAME } from '../../../../common/constants';
 
+const getEnabledInputStreamVars = (packagePolicy: PackagePolicy) => {
+  const enabledInput = packagePolicy.inputs.find((input) => input.enabled);
+  return enabledInput?.streams[0].vars;
+};
+
+const getCspmTelemetryFields = (packagePolicy: PackagePolicy) => {
+  if (packagePolicy.vars?.posture.value !== 'cspm') return;
+
+  const provider = packagePolicy.vars?.deployment?.value;
+  const inputStreamVars = getEnabledInputStreamVars(packagePolicy);
+  const accountType = inputStreamVars?.[`${provider}.account_type`]?.value;
+
+  if (!accountType) return;
```

Comment:
Since for all our old package version we are using single account for aws I would enhance this part: 

if (!accountType && provider === 'aws') return {
    account_type: 'single-account`,
  };